From 699bba9e555aece12b17656152ada929e9fd787d Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Fri, 6 Dec 2019 15:42:10 +0100
Subject: Fix build with Qt 5.14 RC

Add missing includes that are no longer pulled transitively

Differential Revision: https://phabricator.kde.org/D25784
---
 src/audiomixer/mixerwidget.cpp | 1 +
 src/doc/kdenlivedoc.h          | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/audiomixer/mixerwidget.cpp b/src/audiomixer/mixerwidget.cpp
index 08ff50b..d8d6b38 100644
--- a/src/audiomixer/mixerwidget.cpp
+++ b/src/audiomixer/mixerwidget.cpp
@@ -41,6 +41,7 @@
 #include <QSpinBox>
 #include <QDoubleSpinBox>
 #include <QLabel>
+#include <QMouseEvent>
 #include <QStyle>
 #include <QFontDatabase>
 
diff --git a/src/doc/kdenlivedoc.h b/src/doc/kdenlivedoc.h
index 7b56308..64cd7b4 100644
--- a/src/doc/kdenlivedoc.h
+++ b/src/doc/kdenlivedoc.h
@@ -25,6 +25,7 @@
 #ifndef KDENLIVEDOC_H
 #define KDENLIVEDOC_H
 
+#include <QAction>
 #include <QDir>
 #include <QList>
 #include <QMap>
-- 
cgit v1.1
